File: docs/fixtures/core/documents/short_symbol/python/calculate_velocity.py
def calculate_velocity(d: float, duration_s: float) -> float:
    """
    计算平均速度

    Args:
        d: 行进距离
        duration_s: 持续时间
    Returns:
        float: 平均速度
    Raises:
        ValueError: 持续时间不大于零
    """
    velocity_m_per_s = d / duration_s
    return velocity_m_per_s

File: docs/fixtures/core/documents/short_symbol/rust/calculate_velocity.rs
/// 计算平均速度
///
/// # Arguments
/// - d：行进距离
/// - duration_s：持续时间
/// # Returns
/// - 平均速度
/// # Errors
/// - 无
pub fn calculate_velocity(d: f64, duration_s: f64) -> f64 {
    let velocity_m_per_s = d / duration_s;
    velocity_m_per_s
}
